kompose.spec

```diff
@@ -235,20 +234,29 @@ done
 sort -u -o devel.file-list devel.file-list
 %endif
 
+# Why does check need to use buildroot at all? 
 %check
 %if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd):%{gopath}
+export GOPATH=$(pwd):%{gopath}
```

Comment:
Since the ``vendor`` directory breaks the assumption of "all deps are under SOMEPATH/src directory". 

The point of setting ``%{buildroot}`` is to run unit-test over tests that are about to be installed (in rpms). If you run the tests from the $(pwd), it tests what is available in the tarball. Not everything in the tarball is installed in the resulting rpms.

I could test if the vendor is present and move content of the vendor under vendor/src directory. Still, it is a hack.

As long as you know you test what is actually about to be installed in rpms, you can replace ``%{buildroot}`` with ``$(pwd)``.